src/features/calendar/components/index.tsx

```diff
@@ -63,9 +63,19 @@ const Calendar = () => {
   }, [timeScaleStr]);
 
   useEffect(() => {
-    router.query.focusDate = dayjs(focusDate).format('YYYY-MM-DD');
-    router.query.timeScale = selectedTimeScale;
-    router.push(router, undefined, { shallow: true });
+    const focusedDate = dayjs(focusDate).format('YYYY-MM-DD');
+    router.push(
+      {
+        pathname: '/organize/[orgId]/projects/calendar',
```

Comment:
There is a bug here. You have hardcoded the pathname to the global calendar, but there is also a per-project calendar that has stopped working because of this. You need to not hardcode the URL, and either build it from parts (including the campaign ID if one exists) or get the current path from the router as it is right now. If it's not possible to make it `undefined`?